src/handlers/baseFieldsHandlers.ts

```diff
@@ -1,8 +1,35 @@
-import { createBaseField, loadBaseFields, updateBaseField } from '../database';
-import { isTinyPgErrorWithQueryContext, isWritableBaseField } from '../types';
-import { DatabaseError, InputValidationError } from '../errors';
+import { DatabaseError, InputValidationError, NotFoundError } from '../errors';
+import {
+	createBaseField,
+	loadBaseFields,
+	updateBaseField,
+	createOrUpdateBaseFieldLocalization,
+	loadBaseFieldLocalizationsBundle,
+	loadBaseField,
+} from '../database';
+import {
+	isTinyPgErrorWithQueryContext,
+	isValidLanguageTag,
+	isWritableBaseField,
+	isWritableBaseFieldLocalization,
+} from '../types';
 import type { Request, Response, NextFunction } from 'express';
 
+const assertBaseFieldExistsForLocalization = async (
```

Comment:
Right now this does not take any localization context so I think it should be named simply `assertBaseFieldExists`